raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Offences Relating to Records
+(106) Any person who: [a] fails to keep records required to be kept by this Law; [b] fraudulently alters any records kept under this Law; [c] fraudulently makes false statements in any records kept under this Law, commits an offence and is liable on conviction to imprisonment for a period not exceeding 5 years or to a fine not exceeding 10,000 US DOLLARS or to both.
+
+Offences Relating to Restoration Orders
+(107) Any person who fails or refuses to comply with the environmental restoration order made under the order commits an offence and is liable on conviction to imprisonment for a period not exceeding 10 years or to a fine not exceeding 25,000 US DOLLARS or to both.
+
+Liability of Bodies Corporate and Partnerships
+(108) [1] Where an offence is committed under this Law or regulation made under it by a body of persons: [a] in the case of body corporate every director or an officer of the body directly and substantially liable shall also be deemed to be shall be guilty of the offence; and [b] in the case of a partnership every partner or officer of that body directly and substantially liable shall be deemed to be guilty of that offence. [2] a person shall be directly and substantially liable for an offence under this section whether the act is committed by an agent or servant of the body corporate or partnership; [3] an employer shall be liable for the offences committed by an employee, unless he proves that the offence was committed against express or standing directions [4] it shall be a defense for the person deemed liable for the commission of an offence by virtue of subsection [1] if he proves that he exercised all due care and diligence or prevent the commission of the offence having regard to all the circumstances.
+
+Forfeiture, Cancellation, Community Service and Other Orders
+(109) [1] The Court, before which a person is prosecuted for an offence against this Law or any regulations made there under, may, in addition to any other order: [a] upon the conviction of the accused; or [b] if it is satisfied that an offence was committed notwithstanding that no person has been convicted of the offence. [2] Order that the substance, equipment and appliance used in the commission of the offence be forfeited to the Agency and be disposed of as the court directs; [3] In making an order under subsection [1], the court may also order that the cost of disposing of the substance, equipment and appliance referred to in subsection [1], be borne by the accused; [4] The Court may further order the cancellation of any license, permit or other authorization given under this Law and to which the offence relates; [5] The Court may, in addition to any fine it may impose upon an accused person, require him to do community work, which promotes the protection of the environment. [6] The Court may also issue an environmental restoration order against the accused in accordance with Part VIII of this Law.
```

Comment:
Please replace each semi-colon preceding a number in square brackets with a period.